module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php

```diff
@@ -339,6 +339,67 @@ public function testGetMergedConfigNoKey()
         );
     }
 
+    /**
+     * Test getMergedConfig() for internal mixins
+     *
+     * @return void
+     */
+    public function testGetMergedConfigGetMixinFromVendorPackage()
+    {
+        $ti = $this->getThemeInfo();
+        $config = $ti->getMergedConfig(
+            'module/VuFindTheme/tests/fixtures/vendor/example/res/theme/mixin.config.php'
```

Comment:
As noted above, I don't think getMergedConfig is supposed to work like this, so this test might need revision.